src/aead/block.rs

```diff
@@ -78,14 +79,56 @@ impl Block {
 impl From<&'_ [u8; BLOCK_LEN]> for Block {
     #[inline]
     fn from(bytes: &[u8; BLOCK_LEN]) -> Self {
-        unsafe { core::mem::transmute_copy(bytes) }
+        Self {
+            subblocks: [
+                u64::from_le_bytes(
```

Comment:
Why do you use `u64::from_le_bytes` instead of `u64::from_be_bytes`?

Reply:
Saw both `Block::from_le_bytes` and `Block::from_be_bytes` above, so coin flip. Do you prefer `u64::from_be_bytes` / is that more correct?

Reply:
This was my mistake. After looking at the BoringSSL AES impl, see it uses big-endian for keys. Updated with a fix.